microsim/bp_treatment_strategies.py

```diff
@@ -47,11 +47,18 @@ def get_changes_vectorized(self, x):
         x.dbpNext = x.dbpNext - AddASingleBPMedTreatmentStrategy.dbpLowering
         return x
 
-    def rollback_changes_vectorized(self, x):
-        x.sbpNext = x.sbpNext + AddASingleBPMedTreatmentStrategy.sbpLowering
-        x.dbpNext = x.dbpNext + AddASingleBPMedTreatmentStrategy.dbpLowering
-        x.bpMedsAddedNext = 0
-        x.totalBPMedsAddedNext = 0
+    def rollback_changes_on_current_values_vectorized(self, x):
```

Comment:
i suspect that we can get rid of the rollback_changes_vectorized() and just keep the rollback_changes_on_current_values() method